src/server/scripts/Commands/cs_server.cpp

```diff
@@ -267,57 +267,57 @@ class server_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleServerShutDownCommand(ChatHandler* /*handler*/, int32 time, Optional<int32> exitCode)
+    static bool HandleServerShutDownCommand(ChatHandler* /*handler*/, int32 time, Optional<std::string> reason, Optional<int32> exitCode)
```

Comment:
```suggestion
    static bool HandleServerShutDownCommand(ChatHandler* /*handler*/, int32 time, Optional<int32> exitCode, Tail reason)
```

Reply:
@Nyeriah need use `Tail`. This argument must necessarily be at the end, because it covers the whole text